score/concurrency/BUILD

```diff
@@ -467,3 +471,71 @@ cc_unit_test_suites_for_host_and_qnx(
         "//visibility:public",
     ],
 )
+
+
+# ############################################################
+# SAFETY ELEMENT OUT OF CONTEXT
+# ############################################################
+sphinx_docs_library(
+    name = "assumptions_of_use",
+    srcs = ["design/assumptions_of_use.md"],
+)
+
+sphinx_docs_library(
+    name = "component_requirements",
+    srcs = ["design/component_requirements.md"],
+)
+
+sphinx_docs_library(
+    name = "architectural_design",
+    srcs = ["design/architectural_design.md"],
+)
+
+sphinx_docs_library(
+    name = "safety_analysis",
+    srcs = ["design/safety_analysis.md"],
+)
+
+safety_element_out_of_context(
```

Comment:
Can you show how this would be used? e.g. by a different module or by integration?